### PR TITLE
React crud

### PR DESCRIPTION
**Pull request by chiragthakuri**
Hello @abhi886 @ajeetchaulagain ! Could you please look into the code and suggest if there is anyway it can be further optimized? Thank you! 

P.S you need to refresh the page to seed the newly added data. I will FIX this after implementing CONTEXT API.

### CODE REVIEW

**Comment by abhi886** [comment]
@chiragthakuri everything looks good to me . Any comments @ajeetchaulagain..??

File: frontend/src/pages/Home.js [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+      const response = await fetch('/api/numbers/');
+      const json = await response.json();
+
+      if (response.ok) {
+        setNumbers(json);
+      }
+    };
+
+    fetchNumbers();
+  }, []);
```
Comment (ajeetchaulagain) [inline frontend/src/pages/Home.js:19]: Would be nice to have minimal error handling with try/catch here. For this context its not handling 500 error.

File: frontend/src/components/NumberForm.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    });
+
+    const json = await response.json();
+
+    if (!response.ok) {
+      setError(json.error);
+    }
+
+    if (response.ok) {
+      setError(null);
```
Comment (ajeetchaulagain) [inline frontend/src/components/NumberForm.js:29]: setError is not needed here as its state is never updated.

File: frontend/src/components/NumberForm.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+
+const NumberForm = () => {
+  const [phone_number, setPhoneNumber] = useState('');
+  const [country_of_origin, setCountry] = useState('');
+  const [description, setDesc] = useState('');
+  const [error, setError] = useState(null);
```
Comment (ajeetchaulagain) [inline frontend/src/components/NumberForm.js:7]: 4 useState 🤔 - This would be a perfect scenario to use a reducer.

**Review by ajeetchaulagain — APPROVED** [review]
Hey @chiragthakuri @abhi886  Most of the changes LGTM with few minor comments and possible improvements (Leaving upto you guys to decide if those changes are in scope of this PR or not as per the plan)

**Comment by chiragthakuri** [comment]
Thanks for your insight @abhi886  and @Ajeet. Indeed, the changes that you mentioned regarding reducer and errorhandling are not in scope of this PR and will be added in the next development phase.

**Pull request merged**